src/construction.cpp

```diff
@@ -414,12 +414,29 @@ void construction_menu()
 
                         current_line.str( "" );
                         // display required skill and difficulty
-                        int pskill = g->u.get_skill_level( current_con->skill );
-                        int diff = ( current_con->difficulty > 0 ) ? current_con->difficulty : 0;
+                        if( current_con->required_skills.empty() ) {
+                            current_line << _( "N/A" );
+                        } else {
+                            current_line <<
+                                enumerate_as_string( current_con->required_skills.begin(),
+                                                     current_con->required_skills.end(),
+                                                     []( const std::pair<skill_id, int> &skill ) {
+                                nc_color col;
+                                int s_lvl = g->u.get_skill_level( skill.first );
+                                if( s_lvl < skill.second ) {
+                                    col = c_red;
+                                } else if( s_lvl < skill.second * 1.25 ) {
+                                    col = c_ltblue;
+                                } else {
+                                    col = c_white;
+                                }
+
+                                std::string color_s = "<color_" + string_from_color( col ) + ">";
+                                return string_format( "%s%s (%d)</color>", color_s.c_str(),
```

Comment:
I'd rather see it as `get_color_tag( col )`, but went with low number of changes here.